core-bundle/src/Resources/contao/models/PageModel.php

```diff
@@ -102,6 +102,7 @@
  * @property string  $templateGroup
  * @property string  $enforceTwoFactor
  * @property integer $twoFactorJumpTo
+ * @property string  $mailer_transport
```

Comment:
Please also adjust the `find*()` and `count*()` methods.

Reply:
Done.